api/preprints/serializers.py

```diff
@@ -73,71 +73,73 @@ class PreprintSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
     id = IDField(source='_id', read_only=True)
     type = TypeField()
 
-    date_created = VersionedDateTimeField(source='created', read_only=True)
+    date_created = VersionedDateTimeField(source='created', allow_null=True, read_only=True)
     date_modified = VersionedDateTimeField(source='modified', read_only=True)
     date_published = VersionedDateTimeField(read_only=True)
     original_publication_date = VersionedDateTimeField(required=False)
     doi = ser.CharField(source='article_doi', required=False, allow_null=True)
-    is_published = ser.BooleanField(required=False)
-    is_preprint_orphan = ser.BooleanField(read_only=True)
-    license_record = NodeLicenseSerializer(required=False, source='license')
     title = ser.CharField(source='node.title', required=False)
     description = ser.CharField(required=False, allow_blank=True, allow_null=True, source='node.description')
-    tags = JSONAPIListField(child=NodeTagField(), required=False, source='node.tags')
-    node_is_public = ser.BooleanField(read_only=True, source='node__is_public')
-    preprint_doi_created = VersionedDateTimeField(read_only=True)
+    date_retracted = VersionedDateTimeField(read_only=True, allow_null=True)
+    retraction_justification = HideIfNotWithdrawal(ser.CharField(required=False, allow_blank=True))
```

Comment:
It shouldn't affect anything, but for posterity/correctness this should also be `read_only`